ios/keyman/Keyman/Keyman.xcodeproj/xcshareddata/xcschemes/SWKeyboard.xcscheme

```diff
@@ -43,6 +43,11 @@
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES"
       launchAutomaticallySubstyle = "2">
+      <RemoteRunnable
```

Comment:
What is this for?

Reply:
Slipped in; that's the most recent 'debug' setting I used for the SWKeyboard.  We used to not get these because of .gitignore, but I removed that setting when starting to work on iOS unit testing - it caused necessary data for running the unit tests elsewhere (like on the build agents) to not be pushed.